M2/Macaulay2/e/util.cpp

```diff
@@ -2,7 +2,17 @@
 
 #include "stdinc-m2.hpp"
 #include "util.hpp"
-#include <string>
+
```

Comment:
I think `#include <string>` should still appear here, since this file uses it. (It should appear in `util.hpp` as well)